src/Tools/dotnet-format/dotnet-format.runtimeconfig.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.1",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "2.1.2"
+    }
+  }
```

Comment:
Why are we hard coding this versus letting the SDK generate it?

Reply:
`dotnet pack` was not generating it for me and I was getting the following error:

```
A fatal error was encountered. The library 'hostpolicy.dll' required to execute the application was not found in 'C:\Program Files\dotnet'.
```

After searching a bit more I found that the Imports.targets file was setting `<GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>` for me. =)